src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -1043,6 +1047,7 @@ ActStatus ActionHandlers::check_and_send_emm_info(SM::ControlBlock& cb)
     	temp.enb_s1ap_ue_id = ue_ctxt->getS1apEnbUeId();
     	temp.mme_s1ap_ue_id = ue_ctxt->getContextID();
     	temp.dl_seq_no = ue_ctxt->getDwnLnkSeqNo();
+    	ue_ctxt->setDwnLnkSeqNo(temp.dl_seq_no+1);
```

Comment:
what is impact of dl_seq_no on the security. Do we allow multiple messages to be in pipeline towards UE ?

Reply:
Can you give more context on this query? It is possible to send multiple messages towards UE.